content/en/docs/tasks/manage-kubernetes-objects/kustomization.md

```diff
@@ -105,18 +113,19 @@ EOF
 The generated ConfigMap can be examined with the following command:
 
 ```shell
-kubectl kustomize ./
+BAZ=Qux kubectl kustomize ./
 ```
 
 The generated ConfigMap is:
 
 ```yaml
 apiVersion: v1
 data:
-    FOO=Bar
+  BAZ: Qux
+  FOO: Bar
 kind: ConfigMap
 metadata:
-  name: example-configmap-1-8mbdf7882g
+  name: example-configmap-1-892ghb99c8
```

Comment:
nit: do we need to change this too?

Reply:
This is the encoded SHA256 of the configmap [generated by kustomize](https://github.com/kubernetes/cli-runtime/blob/v0.20.12/pkg/kustomize/k8sdeps/transformer/hash/hash.go#L162). So yes, I believe it should change to reflect the output a user running the examples would see.